acdc-core/src/main/scala/com/salesforce/mce/acdc/db/DatasetInstanceQuery.scala

```diff
@@ -66,6 +66,10 @@ object DatasetInstanceQuery {
       .result
   }
 
+  def expire() = {
+    DatasetInstanceTable().filter(_.updatedAt < LocalDateTime.now().minusDays(90)).delete
```

Comment:
Instead of 90, let's make the days configurable

Reply:
as environment variable or in conf file is enough?

Reply:
If you put it as config, it would work with env too.

Reply:
like this https://github.com/salesforce/acdc/pull/42/commits/2395f95a43bbef9bc178f6803459dc708729ff86?